Userland/Libraries/LibGfx/Font/OpenType/Font.h

```diff
@@ -90,6 +92,22 @@ class Font : public Gfx::VectorFont {
     Optional<Kern> m_kern;
     Optional<Fpgm> m_fpgm;
     Optional<Prep> m_prep;
+
+    // This cache stores information per code point.
+    // It's segmented into pages with data about 256 code points each.
+    struct GlyphPage {
+        static constexpr size_t glyphs_per_page = 256;
+
+        u16 glyph_ids[glyphs_per_page];
```

Comment:
`u32 glyph_ids`? That's the type returned by Cmap

Reply:
Yeah it should be `u32`, I don't know why my brain wanted a `u16` :thonk: